lib/matplotlib/tri/triangulation.py

```diff
@@ -12,19 +12,23 @@ class Triangulation(object):
 
     Parameters
     ----------
-    x, y : array_like of shape (npoints)
+    x, y : array-like of shape (npoints)
         Coordinates of grid points.
     triangles : integer array_like of shape (ntri, 3), optional
         For each triangle, the indices of the three points that make
         up the triangle, ordered in an anticlockwise manner.  If not
         specified, the Delaunay triangulation is calculated.
-    mask : boolean array_like of shape (ntri), optional
+    mask : boolean array-like of shape (ntri), optional
         Which triangles are masked out.
 
     Attributes
     ----------
-    `edges`
-    `neighbors`
+    edges : int array of shape (nedges, 2)
+        See `~.Triangulation.edges`
```

Comment:
This line is throwing an error for me when building the docs locally. 

`d:\...\matplotlib\lib\matplotlib\tri\triangulation.py:docstring of matplotlib.tri.Triangulation:40:
toctree references unknown document 'api/matplotlib.tri.Triangulation.edges'`

I wonder anyways why 
```
   edges : int array of shape (nedges, 2)
        See `~.Triangulation.edges`
```
translates into 

![image](https://user-images.githubusercontent.com/23121882/42727914-42e8f914-87af-11e8-81c8-c12e52a19da6.png)

see [link to built doc](https://11142-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/tri_api.html)

Is there some novel doc interpolation going on? Does this have any additional requirement?

Reply:
The built documentation seems to be pulled from the docstring on the `edges` property below.  The issues maybe different versions of numpydoc behaving differently...

Reply:
Good catch, thanks for the hint. I update the requirement in https://github.com/matplotlib/matplotlib/pull/11667